plugin/src/main/less/workflow-editor.less

```diff
@@ -2,14 +2,226 @@
   z-index: 8 !important; /* jenkins form buttons have a z-index of 9 */
   position: absolute;
   bottom: 0;
-  background-color: #f0f0f0;
-  background-color: var(--medium-grey, #f0f0f0);
-  width: 100%;
-  height: 0.35rem;
-  cursor: row-resize;
-  opacity: 0.7;
-}
-.workflow-editor-wrapper .ui-resizable-handle:hover {
-  background-color: var(--primary, #f0f0f0);
-  opacity: 1;
+  right: 0;
+  cursor: se-resize;
```

Comment:
This change is inconsistent with regular textarea (e.g., schedule periodically) resize cursors at least on MacOS.

Reply:
I agree - was for consistency with textareas in Chrome/Safari, at least from the ones I've tried. Makes more sense anyway to have it just vertical.

Reply:
Ah, that explains it, I did not consider this to be browser-dependent.